packages/legend-extension-dsl-serializer/src/models/protocols/pure/DSLSerializer_PureProtocolProcessorPlugin.ts

```diff
@@ -148,7 +149,13 @@ export class DSLSerializer_PureProtocolProcessorPlugin extends DSLMapping_PurePr
           );
           const element = getSchemaSet(path, context.graph);
           element.format = elementProtocol.format;
-          element.schemas = elementProtocol.schemas;
+          element.schemas = elementProtocol.schemas.map((schema) => {
+            const schemaElement = new Schema();
```

Comment:
You lack the assertions here, you should refer to how we do it here - https://github.com/finos/legend-studio/pull/557/commits/7ab13787309722be6a58a5ea0f5c4a7ec73c1718

Reply:
Notice how back in #505, we missed a few places where `V1_` objects are assigned to the metamodel which is quite bad.